.github/workflows/test.yaml

```diff
@@ -333,46 +402,21 @@ jobs:
             systems: 'ubuntu-16.04-64 ubuntu-18.04-64'
             tasks: 'tests/...'
             rules: 'main'
-          - group: ubuntu-focal-jammy
-            backend: google
-            systems: 'ubuntu-20.04-64 ubuntu-22.04-64'
-            tasks: 'tests/...'
-            rules: 'main'
-          - group: ubuntu-noble
-            backend: google
-            systems: 'ubuntu-24.04-64'
-            tasks: 'tests/...'
-            rules: 'main'
-          - group: ubuntu-no-lts
+          - group: ubuntu-jammy
             backend: google
-            systems: 'ubuntu-24.10-64'
+            systems: 'ubuntu-22.04-64'
             tasks: 'tests/...'
             rules: 'main'
-          - group: ubuntu-daily
+          - group: 'ubuntu-dev-current'
```

Comment:
I would prefer `ubuntu-development` or `ubuntu-next`, something to that effect, since it's the next release, which is currently in development :)

The `ubuntu-dev-current` and `ubuntu-dev-previous` is confusing to me, IMO.

Reply:
Thinking some more, I do think `ubuntu-daily` is good, as what we're really testing is the ubuntu daily builds.

Reply:
ubuntu-daily or ubuntu-next should be fine for this

Reply:
Done. I changed it back to ubuntu-daily.